centraldogma/src/test/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplierTest.java

```diff
@@ -133,4 +144,106 @@ public void testGetPropertyAbsentName() {
         PropertyDefinition<Object> missingProperty = PropertyDefinition.define("absent.value", Long.class);
         assertFalse(supplier.getProperty(missingProperty).isPresent());
     }
+
+    @Test
+    public void testRegisterWithDefaultSettings() {
+        when(centralDogma.normalizeRevision(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD)).thenReturn(
+                CompletableFuture.completedFuture(Revision.HEAD)
+        );
+        when(centralDogma.listFiles(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD, FILENAME)).thenReturn(
+                CompletableFuture.completedFuture(Collections.emptyMap())
+        );
+        when(centralDogma.push(
+                PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD,
+                String.format("[CentralDogmaPropertySupplier] Property file created: %s",
+                              FILENAME),
+                Change.ofJsonUpsert(FILENAME, defaultProperties()))
+        ).thenReturn(
+                CompletableFuture.completedFuture(
+                        new PushResult(Revision.HEAD, 1)
+                )
+        );
+
+        CentralDogmaPropertySupplier.register(centralDogma, PROJECT_NAME, REPOSITORY_NAME, FILENAME);
+
+        verify(centralDogma, times(1)).normalizeRevision(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD);
+        verify(centralDogma, times(1)).listFiles(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD, FILENAME);
+        verify(centralDogma, times(1)).push(PROJECT_NAME, REPOSITORY_NAME, Revision.HEAD,
+                                            String.format(
+                                                    "[CentralDogmaPropertySupplier] Property file created: %s",
+                                                    FILENAME),
+                                            Change.ofJsonUpsert(FILENAME, defaultProperties()));
+
+    }
+
+    @Test
+    public void testRegisterWithCustomizedSettings() {
+        final Integer settingForPartitionConcurrency = 188;
```

Comment:
Let's stuck with primitive types when there's no point to use a boxed type. `int`

Reply:
I changed the variable type to `int` from `Integer`. https://github.com/line/decaton/pull/134/commits/382f70617ba6fdfb9659f43fc69dd915a89d0c27